### DISCUSSION TITLE
Updating github workflow container digest, but not version comment

### DISCUSSION THREAD

**Discussion by lippoliv** [renovatebot/renovate#26775] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.140.10

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

Renovate updates github actions with digest + comment (`uses: actions/checkout@1e31de5234b9f8995739874a8ce0492dc87873e2 # 4.0.0` -> `uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1`) which is nice. But if I have a container specified for that job, by digest, it doesn't (`container: ghcr.io/renovatebot/renovate@sha256:e21ca1043a6fd060fb58947b2981dae7a1dcd35ec03acf8b4e89ceacdcd2b062 # 37.140.10` -> `container: ghcr.io/renovatebot/renovate@sha256:0269a65171ad71cb1fd7f880d152edbb9d7e1d45013337e46b44a78627255b72 # 37.140.10` but the digest is `37.140.15`)

<img width="1229" alt="image" src="https://github.com/renovatebot/renovate/assets/2561859/106cc1a1-82c8-4438-b028-e0c20799400b">

<img width="1211" alt="image" src="https://github.com/renovatebot/renovate/assets/2561859/a738fbd1-3ac6-4fed-a6cb-edb7592f793e">

Demo-Repo https://git.keine.cloud/oliver/renovate-gh-workflow-container-version-comment

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
Please put the reproduction on github.com

**Reply by lippoliv** [reply]
Thanks for your quick reply. I may get it wrong but from what I understand you ask me to migrate this demo-repo to GitHub.
I reported a problem for the Gitea integration, did you see it? 

What I report probably is unrelated to GitHub / Gitea so I gave it a try to move to GitHub, but this goes hand in hand with re-configuring renovate to github, which I do fail it seems (at least with the time I have right now) (https://github.com/lippoliv/renovate-gh-workflow-container-version-comment/actions/runs/7600616980/job/20699059326), probably because of the container, I don't know. 

If a repo on GitHub really is needed, I would be happy if someone who has more experience with GitHub and Renovate can support me :)

**Reply by rarkins** [reply]
GitHub is needed. You can install the Mend Renovate app instead of self hosting

**Comment by viceice** [comment, marked as answer]
this is expected, the image doesn't need the comment. add the version as container tag instead.

**Reply by viceice** [reply]
`ghcr.io/renovatebot/renovate:37.140.15@sha256:0269a65171ad71cb1fd7f880d152edbb9d7e1d45013337e46b44a78627255b72`

**Reply by viceice** [reply]
https://codeberg.org/forgejo-contrib/forgejo-renovate/src/commit/bc45157d90169a39edea96c0257da702f4658e44/.forgejo/workflows/lint.yml#L10

**Reply by lippoliv** [reply]
funny. I moved on testing arround with other components and if they update. With dockerfiles I did the same notation because I couldn't add comments to `FROM` now it absolutely makes sense to re-use it here.

thanks for opening my eyes :)
